### PR TITLE
gnrc/rpl: Expose configurations to Kconfig

### PR DESCRIPTION
**Pull request by leandrolanzieri**
### Contribution description
This PR moves configuration macros of GNRC RPL module to `CONFIG_` namespace and exposes them to Kconfig.

### Testing procedure
- Examples using rpl should work as usual
- Changing configurations via Kconfig (e.g. using `menuconfig`) should reflect on the configuration macros.

### Issues/PRs references
Part of #12888

### CODE REVIEW

**Comment by cgundogan** [comment]
@leandrolanzieri code-wise the changes look correct. I wanted to give this a test, but the RPL menu does not appear in the menu? Is an include missing?

**Comment by leandrolanzieri** [comment]
> Is an include missing?

Indeed, please give it a try now :-)

**Comment by cgundogan** [comment]
and please squash.

**Comment by leandrolanzieri** [comment]
Done

File: sys/include/net/gnrc/rpl.h [hunk 1]
```diff
@@ -137,8 +138,8 @@ extern "C" {
 /**
  * @brief   Default message queue size to use for the RPL thread.
  */
-#ifndef GNRC_RPL_MSG_QUEUE_SIZE
-#define GNRC_RPL_MSG_QUEUE_SIZE (8U)
+#ifndef CONFIG_GNRC_RPL_MSG_QUEUE_SIZE
+#define CONFIG_GNRC_RPL_MSG_QUEUE_SIZE (8U)
```
Comment (miri64) [inline sys/include/net/gnrc/rpl.h:142]: In #14071 the exponent was used to model the power of 2 restriction of message queues. Should we do this here as well or rather as a follow-up (including the other queue sizes)

Comment (cgundogan) [reply sys/include/net/gnrc/rpl.h:142]: Good point. Let's defer that to a follow-up PR, though. The diff and commit history is already kind of convoluted.

**Pull request merged**